db/fixtures/miq_policy_sets.yml

```diff
@@ -39,6 +39,21 @@
           description: Mark as Non-Compliant
           action_type: default
           options: {}
+      - qualifier: failure
+        failure_sequence: 2
+        failure_synchronous: true
+        MiqEventDefinition:
+          name: containerimage_compliance_check
```

Comment:
Is there a reason why this is `containerimage_compliance_check` and the below action is `container_image_annotate_deny_execution`?  Why not `container_image_compliance_check`?